helper/user_comms_helper.rb

```diff
@@ -1,13 +1,17 @@
 class UserCommsHelper
 
+  require 'yaml'
+  CONFIG = YAML.load_file("config.yml")
```

Comment:
YAML inclusion and file parsing should one take place in one place. Introduce a class which loads the config once and provides its contents.